### ISSUE TITLE
Use EditForm

### ISSUE THREAD

**Issue by fileman** [oqtane/oqtane.framework#110]
There is a reason why you don't use Blazor built-in EditForm in Add and Edit pages?

**Comment by fileman** [comment]
I have tested and no issue found #113

**Comment by sbwalker** [comment]
I began creating Oqtane before the EditForm was introduced. That being said, I have explored the EditForm and these are my observations:

- all of the examples I have seen use <divs> to lay out a form - which results in a vertically oriented form. This is problematic for forms which have many fields as it increases the amount of vertical scrolling required. My preference is a tabular form as it uses screen real estate more efficiently. It is likely possible to achieve this with EditForm but I have not yet seen an example.
- when using EditForm, the form layout code is a lot more verbose and confusing for novice developers. ValidationMessage controls are littered throughout the code even when a field does not require validation. The bind syntax is different than the standard approach and the For= syntax ( ie. "@(() => registerCommand.Email)" ) is not intuitive.
- models become heavier and require more ongoing maintenance
- you still need to validate all input on the server as you cant trust the client validation
- EditForm is not localizable and I can imagine breaking changes in the future in order to make it localizable

I do not really see the clear benefits of using EditForm.
